### PR TITLE
Added enterprise team and repos to config

### PR DESCRIPTION
**Pull request by barakalon**
conf/config.py
* added ```ENTERPRISE``` team and corresponding tag/repos
* alphabetized ```TAGET_REPOS```

### CODE REVIEW

File: conf/config.py [hunk 1]
```diff
@@ -3,14 +3,18 @@
 ML_TEAM_ID = "1654133"
 WEB_TEAM_R1 = "1744555"
 WEB_TEAM_R2 = "1744559"
+ENTERPRISE = "1792247"
 
 TARGET_REPOS = [
+    "admin-api",
```
Comment (oahayder) [inline conf/config.py]: Look at you and your alphabetical order

**Comment by oahayder** [comment]
I'll release this now

File: conf/config.py (hunk 1 above)
Comment (barakalon) [inline conf/config.py]: I'm doing big things with my life, @oahayder

**Comment by barakalon** [comment]
:+1: no rush

**Pull request merged**